master/setup.py

```diff
@@ -444,7 +444,8 @@ def define_plugin_entries(groups):
     'txaio ' + txaio_ver,
     'autobahn ' + autobahn_ver,
     'PyJWT',
-    'distro'
+    'distro',
```

Comment:
we can't make hvac an hard dependency to buildbot. We have to keep it an optionnal dependency.